brave/brave-core
# [Android] Remove legacy swap code

Resolves brave/brave-browser/issues/30542



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

Run through the following parts of wallet and make sure nothing is broken:
1. Tap on any assets, and then tap either "Buy" or "Send".
2. Make sure we can go to the buy sites correctly, and Send Tx can be executed correctly.
3. On top wallet page, go to "Buy" and "Send" using the bottom sheet.
4. Also make sure both buy and send are working correctly.
